praw/__init__.py

```diff
@@ -926,8 +926,9 @@ def get_random_submission(self, subreddit='all'):
         """
         url = self.config['subreddit_random'] % six.text_type(subreddit)
         try:
```

Comment:
Looks like this try block may not be necessary anymore. I'm curious why the RedirectException is no longer being raised. Unless it was always wrong.